Araali-Tools/GoShelly/cobrashelly/aws/aws.go

```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	t "cobrashelly/template"
```

Comment:
What is cobrashelly?

Reply:
Server package name, it took it default as folder name. Will change.